python_modules/dagster-airflow/dagster_airflow/compile.py

```diff
@@ -0,0 +1,28 @@
+import itertools
+
+from collections import defaultdict
+
+
+def _coalesce_solid_order(execution_plan):
+    solid_order = [s.tags['solid'] for s in execution_plan.topological_steps()]
```

Comment:
maybe you only need to pass the `topological_steps()` output here? then you can reuse on L#24 below